logic/minecraft/WorldList.h

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2015 <copyright holder> <email>
```

Comment:
Probably should look over all license headers, I saw 2013-2015 being used in one new file too.